src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRunCodegenNodeFactory.cs

```diff
@@ -222,13 +214,97 @@ public sealed class ReadyToRunCodegenNodeFactory : NodeFactory
                   nameMangler,
                   new ReadyToRunTableManager(context))
         {
-            _importMethods = new Dictionary<TypeAndMethod, IMethodNode>();
 
             Resolver = moduleTokenResolver;
             InputModuleContext = signatureContext;
             CopiedCorHeaderNode = corHeaderNode;
             if (!win32Resources.IsEmpty)
                 Win32ResourcesNode = new Win32ResourcesNode(win32Resources);
+
+            CreateCodegenNodeCaches();
```

Comment:
>CreateCodegenNodeCaches [](start = 12, length = 23)

We can just rename this to CreateNodeCaches, to be consistent with other places

Reply:
Done.